halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/ServiceSettings.java

```diff
@@ -48,6 +50,7 @@
   Map<String, String> env = new HashMap<>();
   String artifactId;
   boolean enabled;
+  boolean auth;
```

Comment:
I suggest including a verb: `authEnabled`, because `isAuth()` reads and looks funny. 

That being said, is this different than SSL? just `auth` or even `authEnabled` doesnt' tell me if authN or authZ has changed.